### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by GrumpyXKitty** [ldtteam/minecolonies#8831] [Q&A]
Hi,Everyone, my minecolonies won't run, because it says that it need Structurize 1.18.2-1.0.424-ALPHA but when I went looking for that I could not find Structurize 1.18.2-1.0.424-ALPHA. Can someone tell me what to do about it?

**Comment by Thodor12** [comment]
Try checking in all versions, if you have the Curseforge app you can find that on the mod page > versions.
If you're looking on the Curseforge website you can go to files > view all (scroll down a bit) and then find the correct version, or go to this link: https://www.curseforge.com/minecraft/mc-mods/structurize/files/all
